### PR TITLE
fetch tutorial word list

### PR DESCRIPTION
**Pull request by OLI9292**

### CODE REVIEW

File: Apiary/Managers/NavigationManager.swift [hunk 1]
```diff
@@ -54,7 +54,7 @@ extension NotificationObservers {
 extension LoadingVCs {
   
   func loadInitialViewController() {
-    let id: StoryboardID = .hive
+    let id: StoryboardID = Datastore.shared.user == nil ? .tutorial : .hive
```
Comment (OLI9292) [inline Apiary/Managers/NavigationManager.swift]: Moving the tutorial check to the initial VC load

File: Apiary/Managers/RealmManager.swift [hunk 2]
```diff
@@ -71,11 +71,11 @@ final class RealmManager {
     }
   }
   
-  static func createUser(userHandle: String) {
+  static func createUser(first: String, last: String, tutorialWords: [TutorialWord]) {
     do {
+      let newUser = User(firstName: first, lastName: last, tutorialWords: tutorialWords)
```
Comment (OLI9292) [inline Apiary/Managers/RealmManager.swift]: little cleaner to add an intializer to user

File: Apiary/ViewControllers/HiveViewController.swift [hunk 3]
```diff
@@ -27,33 +27,11 @@ final class HiveViewController: UIViewController {
...
-          RealmManager.create(words, update: true)
-          self?.layout()
-        }
-      } else {
-        self.store.words = RealmManager.words
-        self.layout()
-      }
+    } else {
+      startSpinner()
+      store.fetchWords() { _ in self.layout() }
```
Comment (OLI9292) [inline Apiary/ViewControllers/HiveViewController.swift]: moved the word list fetch to the Datastore

File: Apiary/ViewControllers/TutorialViewController.swift [hunk 4]
```diff
@@ -5,26 +5,25 @@ import RxSwift
...
-    return views[currentIndex]
-  }
+  let tutorialProgressBar = ProgressBar(type: .expedition)
   
-  var views: [UIView] = [TutorialQuestionFour()]
+  var completion: Variable<Double> = Variable(0.166)
   
-  // Track tutorial progress
-  let tutorialProgressBar = ProgressBar(type: .expedition)
+  var steps = [
```
Comment (OLI9292) [inline Apiary/ViewControllers/TutorialViewController.swift]: found this way a little easier - just deplete this array of views and send notifications to advance

Comment (bencbernstein) [inline Apiary/ViewControllers/TutorialViewController.swift]: sounds good. eventually we could save tutorial progress if it gets long

**Pull request merged**